internal/controller/management/serviceaccounts/serviceaccounts_test.go

```diff
@@ -0,0 +1 @@
+package serviceaccounts
```

Comment:
This particular reconciler should be pretty easy to test. Let's make sure we add some test cases here.